chart/kubeapps/templates/apprepository-crd.yaml

```diff
@@ -1,12 +1,13 @@
-{{- if or (not (.Capabilities.APIVersions.Has "kubeapps.com/v1alpha1")) (.Release.IsUpgrade) -}}
+{{- if not (.Capabilities.APIVersions.Has "kubeapps.com/v1alpha1") -}}
 # The condition above will be true if another instance of Kubeapps is
 # already installed 
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: apprepositories.kubeapps.com
   annotations:
-    "helm.sh/resource-policy": keep
```

Comment:
if I remember correctly, if it's installed with a hook it won't be deleted by helm, isn't it?

Reply:
It wont be deleted no